app/src/utils/utils.ts

```diff
@@ -118,3 +128,31 @@ export function displayResults(
   const end = Math.min(page * perPage, numFound);
   return `${start}-${end} of ${numFound}`;
 }
+
+export const createQueryStringFromObject = (paramObj) => {
+  const params = new URLSearchParams();
+  Object.keys(paramObj).map((name, value) => {
```

Comment:
Two points here:

- `.map` functions return a new array. Use `forEach` instead.
- `value` is not being used so it can be removed. You're iterating over the keys and that's a single value, not a pair.